projects/storefrontlib/src/cms-structure/routing/default-routing-config.ts

```diff
@@ -74,16 +74,16 @@ export const defaultStorefrontRoutesConfig: RoutesConfig = {
 
   // configuration related routes
   configureCPQCONFIGURATOR: {
-    paths: ['configureCPQCONFIGURATOR/:ownerType/entityKey/:entityKey'],
+    paths: ['configure/vc/:ownerType/entityKey/:entityKey'],
```

Comment:
What does `/vc/` mean? I come across this term first time. Does it go for _variant configurator_? I'm in favour of link which are readable. Did we consider `/variant/` or `/cpq/`? Perhaps `/vc/` is also OK and it's only my lack of knowledge. What is most appropriate from the user's as well as technical point of view in your oppinion?

Reply:
'cpq' stands for a different configurator, so we _will_ later have configure/cpq/:ownerType...
And as 'VC' is a well-known term amongst S/4 customers (which we are targeting with that), I think this is fine.